### PR TITLE
fixed the issue of Users filter to fetch corresponding Users to Organisation #701

### PR DESCRIPTION
**Pull request by Sumitmaithani**
**What kind of change does this PR introduce?**
bugfix

**Issue Number:**
Fixes #[701 ](https://github.com/PalisadoesFoundation/talawa-admin/issues/701#event-8947233389)

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**

https://user-images.githubusercontent.com/86047367/230567661-e1da4bd3-3a17-4db2-a492-020acfe43e16.mp4

**If relevant, did you update the documentation?**
Not relevant

**Summary**
The "joinedOrganization" field has been added to user data. It is an array that contains all the organizations that the user has joined. The user's of organizations are now filtered based on which ones match the organization ID.

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by Sumitmaithani** [comment]
@Rumone @ttaylor92 @palisadoes Please review this PR

**Comment by anshgoyalevil** [comment]
@Sumitmaithani 

Please pull the latest code, and re-run the tests.

**Comment by Sumitmaithani** [comment]
@anshgoyalevil 

There is no merging conflict.

**Comment by anshgoyalevil** [comment]
@Sumitmaithani 

Actually, Not because of merge conflicts. There is a new warnings checking file added to the codebase.

If there are warnings in your tests, which were previously passed in CI, they would still fail during PR merge.

So, better to catch them here.

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by Sumitmaithani** [comment]
@palisadoes 

This PR can now be merged.

**Comment by palisadoes** [comment]
@ttaylor92 @Kevoniat @Rumone Please review.

**Comment by Sumitmaithani** [comment]
fixed merged issue

@ttaylor92 @Kevoniat @Rumone Please review.

File: src/screens/OrganizationPeople/OrganizationPeople.test.tsx [hunk 1]
```diff
@@ -493,24 +509,35 @@ describe('Organization People Page', () => {
...
+          window.location.assign('/orgpeople/id=6401ff65ce8e8406b8f07af1');
+          const pathname = window.location.pathname;
+          const id = pathname.split('=')[1];
+          return datas.joinedOrganizations.some((org) => org._id === id);
+        }
+      ).length;
+
+      await wait();
+      const expectedUsersLength = screen.getAllByTestId('orgpeoplelist').length;
+      expect(expectedUsersLength).toBe(totalNumPeople);
```
Comment (Rumone) [inline src/screens/OrganizationPeople/OrganizationPeople.test.tsx:540]: `expectedUsersLength` is a variable to store the amount of users rendered in the test. The variable `totalNumPeople` would be best to switch what it stores with `expectedUsersLength`. That way the code reasons

`expect(totalNumPeople).toBe(expectedUsersLength)`

**Review by Rumone — CHANGES_REQUESTED** [review]

File: src/screens/OrganizationPeople/OrganizationPeople.test.tsx (hunk 1 above)
Comment (Sumitmaithani) [reply src/screens/OrganizationPeople/OrganizationPeople.test.tsx]: @Rumone Thanks for review... I have implemented the changes..

**Comment by Sumitmaithani** [comment]
@ttaylor92 @Kevoniat @Rumone @palisadoes Please review this PR. I m working on this [issue ](https://github.com/PalisadoesFoundation/talawa-admin/issues/556)for that this need to be merge

**Review by Rumone — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**